src/groovy/mama/ng/scheduler/CronDefinition.groovy

```diff
@@ -0,0 +1,68 @@
+package mama.ng.scheduler
+
+import java.util.regex.Matcher
+import java.util.regex.Pattern
+
+class CronDefinition {
+
+    List<Integer> minutes
+    List<Integer> hours
+    List<Integer> daysOfMonth
+    List<Integer> months
+    List<Integer> daysOfWeek
+
+    int minute = 1
+    int hour = 7
+    int dayOfMonth = 17
+    int month = 29
+    int dayOfWeek = 39
+
+    final static String REGEX = '^(((([1-5]?[0-9])(\\,|\\-))*([1-5]?[0-9]))|\\*)\\ ' + //min (0 - 59)
```

Comment:
That sucks a bit. We could make things very slightly nicer using the `/foo/` string format to avoid some slashes, e.g. `/^foo\*/` vs `'^foo\\*'` but it's not a big win. I'm wondering whether we shouldn't abandon using regular expressions here and use a simple parsing function instead (a tokenizer would allow us to split on spaces and after that it's easy to use a much simpler regular expression to check the individual tokens)?